introduction.md

```diff
@@ -1,5 +1,12 @@
 Luke, I am your grandfather.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?